site/faq.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Weave Net FAQ
+layout: default
+---
+
+
+
+###Q:How do I obtain the IP of a specific container when I'm using Weave?
+
+Use `weave ps` to see which address was allocated to a container on a Weave network.  
+
+See [Address Allocation](/site/features/address-allocation.md).
+
+
+###Q: How do I expose one of my containers to the outside world?
+
+Exposing a container to the outside world is described in [Exposing Services to the Outside](/site/using-weave/service-export.md).
+
+
+###Q: My dockerized app needs to check the request of a remote IP. Can I manually change the IP of a container?
```

Comment:
I don't understand this question.  What does changing the IP have to do with "check the request" ?  What does that even mean?